test/adapters/postgres.test.js

```diff
@@ -1,13 +1,11 @@
+"use strict";
+
 var it = require('it'),
     assert = require('assert'),
     patio = require("../../lib"),
     sql = patio.SQL,
-    comb = require("comb-proxy"),
-    config = require("../test.config.js"),
```

Comment:
how do i run the tests now?  looks like patio_db needs setting somehow in the ENV.